draft-ietf-rats-corim.md

```diff
@@ -1080,84 +1080,112 @@ The verification procedure is divided into three separate phases:
 * Evidence collection
 * Evidence appraisal
 
-At a few well-defined points in the procedure, the Verifier behaviour will depend on the specific CoRIM profile.
-Each CoRIM profile MUST provide a description of the expected Verifier behavior for each of those well-defined points.
+At a few well-defined points in the procedure, the Verifier behaviour will
+depend on the specific CoRIM profile.
+Each CoRIM profile MUST provide a description of the expected Verifier behavior
+for each of those well-defined points.
 
 Note that what follows describes a simplified implementation.
-Verifiers claiming compliance with this specification must exhibit the same externally visible behavior as described here,
+Verifiers claiming compliance with this specification must exhibit the same
+externally visible behavior as described here,
 they are not required to use the same internal data structures.
-For example, it is expected that the resources used during the initialisation phase can be amortised across multiple appraisals.
+For example, it is expected that the resources used during the initialisation
+phase can be amortised across multiple appraisals.
 
 ## Appraisal Context initialisation
 
-The goal of the initialisation phase is to load the CoRIM Appraisal Context with objects such as CoRIM tags,
-cryptographic validation key material (e.g., raw public keys, root certificates, intermediate CA certificate chains), etc.
-that will be used in the subsequent Evidence Appraisal phase.
+The goal of the initialisation phase is to load the CoRIM Appraisal Context
+with objects such as CoRIM tags,
+cryptographic validation key material (e.g., raw public keys, root certificates,
+intermediate CA certificate chains), etc. that will be used in the subsequent
+Evidence Appraisal phase.
 
 ### CoRIM Selection
 
 All available CoRIMs are collected.
-A Verifier may be pre-configured with a large number of tags describing many types of device.
-All CoRIMs are loaded at this stage, later stages will select the CoRIMs appropriate to the Evidence Appraisal step.
+A Verifier may be pre-configured with a large number of tags describing many
+types of device.
+All CoRIMs are loaded at this stage, later stages will select the CoRIMs
+appropriate to the Evidence Appraisal step.
 
-CoRIMs that are not within their validity period, or that cannot be associated with an authenticated and authorised source MUST be discarded.
+CoRIMs that are not within their validity period, or that cannot be associated
+with an authenticated and authorised source MUST be discarded.
 
-CoRIM that are secured by a cryptographic mechanism such as a signature which does not pass validation MUST be discarded.
+CoRIM that are secured by a cryptographic mechanism such as a signature which
+does not pass validation MUST be discarded.
```

Comment:
this seems to repeat L1112, specifically:

_"[...] that cannot be associated with an authenticated [...] source MUST be discarded."_